src/select/select.js

```diff
@@ -177,14 +177,15 @@ angular.module('mgcrea.ngStrap.select', ['mgcrea.ngStrap.tooltip', 'mgcrea.ngStr
 
         $select.$onKeyDown = function(evt) {
           if (!/(9|13|38|40)/.test(evt.keyCode)) return;
-          evt.preventDefault();
-          evt.stopPropagation();
 
           // release focus on tab
           if (options.multiple && evt.keyCode === 9) {
             return $select.hide();
           }
 
+          evt.preventDefault();
+          evt.stopPropagation();
```

Comment:
This fix is incorrect since tab won't work properly if options.multiple === false